### ISSUE TITLE
how run 'npm ci` instead 'npm i'

### ISSUE THREAD

**Issue by shurygindv** [mjeanroy/node-maven-plugin#144]
Hi, how the plugin may run `npm ci` commands with arguments instead just `npm i`? 

Thank you a lot!

**Comment by mjeanroy** [comment]
Hi @shurygindv,

You should be able to override the `install` script run during the `install` goal, for example:

```xml
<plugin>
  <groupId>com.github.mjeanroy</groupId>
  <artifactId>node-maven-plugin</artifactId>
  <version>0.7.2</version>
  <configuration>
    <installScript>ci</installScript>
  </configuration>
  <executions>
    <execution>
      <id>front:build</id>
      <goals>
        <goal>check</goal>
        <goal>install</goal>
        <goal>build</goal>
        <goal>test</goal>
      </goals>
    </execution>
  </executions>
</plugin>
```

Let me know if that works, otherwise I'll fix it ;)

**Comment by shurygindv** [comment]
It almostly worked! But in some time it threw an error (use your code snippet above) 

`[ERROR] Cannot execute npm run ci command: it is not defined in package.json (please check file: ../package.json).`

**Comment by mjeanroy** [comment]
@shurygindv I see, I just fixed it, please update to version `0.9.1`, it should works ;)
